### PR TITLE
[doc]Stream load doc fix

### PR DESCRIPTION
**Pull request by hf200012**
# Proposed changes

Issue Number: close #xxx

## Problem Summary:

Describe the overview of changes.

## Checklist(Required)

1. Does it affect the original behavior: (Yes/No/I Don't know)
2. Has unit tests been added: (Yes/No/No Need)
3. Has document been added or modified: (Yes/No/No Need)
4. Does it need to update dependencies: (Yes/No)
5. Are there any changes that cannot be rolled back: (Yes/No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by JNSimba — DISMISSED** [review]
LGTM

File: docs/zh-CN/docs/sql-manual/sql-reference/Show-Statements/SHOW-STREAM-LOAD.md [hunk 1]
```diff
@@ -50,6 +50,7 @@ SHOW STREAM LOAD
 
 说明：
 
+1. 默认 BE 是不记录 Stream Load 的记录，如果你要查看需要再 BE 上启用记录，`enable_stream_load_record=true`，这里需要重启BE
```
Comment (caiconghui) [inline docs/zh-CN/docs/sql-manual/sql-reference/Show-Statements/SHOW-STREAM-LOAD.md:53]:
```suggestion
1. 默认 BE 是不记录 Stream Load 的记录，如果你要查看需要在BE 上启用记录，`enable_stream_load_record=true`，这里需要重启BE
```

**Comment by morningman** [comment]
After the website upgraded, we should find a way to update the document on website.
Currently, it will not update automatically.

File: docs/zh-CN/docs/sql-manual/sql-reference/Show-Statements/SHOW-STREAM-LOAD.md (hunk 1 above)
Comment (hf200012) [reply docs/zh-CN/docs/sql-manual/sql-reference/Show-Statements/SHOW-STREAM-LOAD.md]: solved

**Comment by hf200012** [comment]
> After the website upgraded, we should find a way to update the document on website. Currently, it will not update automatically.

I will update the website synchronously

**Review by JNSimba — APPROVED** [review]
LGTM

**Pull request closed without merging**